database/index.js

```diff
@@ -6,12 +6,64 @@ const pool = new Pool({
 });
 
 module.exports = {
-  getOne: (callback) => {
-    pool.query('SELECT * FROM reviews ORDER BY id ASC', (error, datas) => {
+  getAll: (id, callback) => {
+    pool.query(`SELECT * FROM reviews WHERE roomId = ${id} ORDER BY id ASC`, (error, datas) => {
       if (error) {
         throw error;
       }
       callback(null, datas.rows);
     });
   },
+
+  getOne: (ids, callback) => {
+    // console.log(ids);
+    //{ roomId: '210329', reviewId: '746820' }
+    pool.query(`SELECT * FROM reviews WHERE roomId = ${ids.roomId} AND id = ${ids.reviewId} ORDER BY id ASC`, (error, datas) => {
+      if (error) {
+        throw error;
+      }
+      // console.log(datas.rows);
+      let findUser = datas.rows[0];
+      // console.log(findUser.userid);
+      pool.query(`SELECT * FROM users INNER JOIN reviews ON reviews.userid = users.id WHERE users.id = ${findUser.userid} AND reviews.id = ${ids.reviewId}`, (err, data) => {
+        if (error) {
+          throw error;
+        }
+        // console.log(data);
+        callback(null, data.rows);
+      })
+    });
+  },
+
+  create: (element, callback) => {
+    // console.log(element.body);
+    const text = 'INSERT INTO reviews(id, reviewtext, reviewdate, accuracy, location, communication, checkin, cleanliness, value, hostname, hostpic, responsedate, responsetext, roomid, userid) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15) RETURNING *'
```

Comment:
I think you can leave the values as question marks in the parenthesis and it will be filled in with the variables

Reply:
I see! I will try that!